.editorconfig

```diff
@@ -0,0 +1,25 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
Maybe we should leave this unspecified to keep Windows folks happy?

Reply:
I guess they're used to this problem, I checked a bunch of project settings they all af this setting: https://github.com/editorconfig/editorconfig/wiki/Projects-Using-EditorConfig

Reply:
More seriously, cross-platform projects seems to omit that settings, see https://github.com/Microsoft/vscode/blob/master/.editorconfig (also interesting for how they group targets with common settings)